lib/active_resource/base.rb

```diff
@@ -1721,7 +1721,7 @@ class Base
     extend ActiveModel::Naming
     extend ActiveResource::Associations
 
-    include Callbacks, CustomMethods, Validations
+    include Callbacks, Validations
```

Comment:
We need a `DeprecatedCustomMethods` that has the current implementation of `CustomMethods` but show a deprecation warning when the methods are called.

Reply:
Should `DeprecatedCustomMethods` just be copy-pasted? Because I feel like it is the only way, but this is not ok in case of future changes in `CustomMethods`

Reply:
yeah, copy and pasted. It is only temporary anyway.

Reply:
Okay, I'll try my best on starting it soon